applications/nrf5340_audio/dfu/conf/overlay-mcuboot.conf

```diff
@@ -8,7 +8,6 @@ CONFIG_SIZE_OPTIMIZATIONS=y
 CONFIG_MAIN_STACK_SIZE=20480
 CONFIG_MBEDTLS_CFG_FILE="mcuboot-mbedtls-cfg.h"
 CONFIG_PCD_APP=y
-CONFIG_PARTITION_MANAGER_ENABLED=y
```

Comment:
We use Kconfig to decide which overlay should be loaded by mcuboot in cmake file.
This has to place before zephyr bookplate kicking in.
This macro in nrf/cmake/multi_image.cmake can't not be applied.
>   add_overlay_config(
>     ${ACI_NAME}
>     ${NRF_DIR}/subsys/partition_manager/partition_manager_enabled.conf
>     )
> 
We have to explicitly select CONFIG_PARTITION_MANAGER_ENABLED in mcuboot overlay of our project.
Otherwise mcuboot won't boot application.
If CONFIG_PARTITION_MANAGER_ENABLED is not allowed to be set by application, I propose another commit to include partition_manager_enabled.conf overlay for not breaking LE Audio application DFU.